TEXT/main.py

```diff
@@ -0,0 +1,38 @@
+import pygame, sys

```

Comment:
This file can probably be removed from the main branch. 

Git will handle version control, you can [retrieve past versions of files on an as-needed basis](http://stackoverflow.com/questions/953481/find-and-restore-a-deleted-file-in-a-git-repository), so you don't have to use folders to separate files you think you don't need anymore.